alcaval_steps.py

```diff
@@ -116,3 +128,27 @@
 steps['HARVEST_CRUZET2021'] = merge([ {'--scenario':'cosmics'}, steps['HARVESTDefault'] ])
 steps['HARVEST_MWGR2022'] = merge([ {'--scenario':'cosmics'}, steps['HARVESTDefault'] ])
 steps['HARVEST_CRAFT22'] = merge([ {'--scenario':'cosmics'}, steps['HARVESTDefault'] ])
+steps['HARVEST_CRAFT22_v2'] = merge([ {'--scenario':'cosmics',
+				'--customise': 'Configuration/DataProcessing/RecoTLR.customiseCosmicData',
+				}, steps['HARVESTDefault'] ])
+
+# Collision 2022
+steps['RunHLTPhysics2022A']={'INPUT':InputInfo(dataSet='/HLTPhysics/Run2022A-v1/RAW',ls={352567: [[1,126]]})}
+steps['RunJetHT2022A']={'INPUT':InputInfo(dataSet='/JetHT/Run2022A-v1/RAW',ls={352567: [[1,126]]})}
+steps['RunZeroBias2022A']={'INPUT':InputInfo(dataSet='/ZeroBias/Run2022A-v1/RAW',ls={352567: [[1,126]]})}
+steps['RunMinimumBias2022A']={'INPUT':InputInfo(dataSet='/MinimumBias/Run2022A-v1/RAW',ls={352567: [[1,126]]})}
+
+steps['RunHLTPhysics2022A_v2']={'INPUT':InputInfo(dataSet='/HLTPhysics/Run2022A-v1/RAW',ls={353060: [[1,500]]})}
+steps['RunZeroBias2022A_v2']={'INPUT':InputInfo(dataSet='/ZeroBias/Run2022A-v1/RAW',ls={353060: [[1,500]]})}
+steps['RunMinimumBias2022A_v2']={'INPUT':InputInfo(dataSet='/MinimumBias/Run2022A-v1/RAW',ls={353060: [[1,500]]})}
+
+steps['HLT_Collision22_v1'] = merge( [ {
+                '-s': 'L1REPACK:uGT,HLT',
+                '--customise': 'L1Trigger/Configuration/customiseUtils.L1TGlobalMenuXML'
+                }, step2Defaults] )
+steps['RECO_Collision22_v1']=merge([{
+                            '-s'            : 'RAW2DIGI,L1Reco,RECO,DQM'},
+                            {'--procModifiers'  : 'siPixelQualityRawToDigi'},
+                            {'--customise':'Configuration/DataProcessing/RecoTLR.customisePostEra_Run3,RecoLocalCalo/Configuration/customiseHBHEreco.hbheUseM0FullRangePhase1'},
```

Comment:
@tvami , all,
Please leave comments if you think any changes to current configuration is required.

Reply:
I think it should be good, but I'm uncertain about `hbheUseM0FullRangePhase1` I'll ask HCAL

Reply:
I will keep this configuration until then